renderdoc/driver/gl/gl_debug.cpp

```diff
@@ -3725,7 +3725,8 @@ void GLReplay::InitPostVSBuffers(uint32_t eventID)
   }
 
   // get buffer data from buffer attached to feedback object
-  float *data = (float *)gl.glMapNamedBufferEXT(DebugData.feedbackBuffer, eGL_READ_ONLY);
+  float *data = (float *)gl.glMapBufferRange(eGL_TRANSFORM_FEEDBACK_BUFFER, 0,
```

Comment:
I see yes I can move the glMapBufferRange call into gl_emulated.cpp::_glMapNamedBufferEXT, but for some reason this gl.glMapNamedBufferEXT call is going into WrappedOpenGL::glMapNamedBufferEXT, and not into the emulated function.